source/events/events.rst

```diff
@@ -0,0 +1,295 @@
+.. include:: ../references.txt
+
+.. iact_events:
+
+``EVENTS`` extension
+====================
+
+The ``EVENTS`` extension is a binary FITS table that contains an event list.
+Each row of the table provides information that characterises one event.
+The mandatory and optional columns of the table are listed below. In
+addition, a list of header keywords providing metadata is specified.
+Also here there are mandatory and optional keywords.
+The suggested extension name of the binary table is ``EVENTS``.
+
+
+Mandatory columns
+-----------------
+
+* ``EVENT_ID`` tform: ``1J``
+    * Event identification number at the DL3 level
+      (lower data levels could be different, see note below).
+* ``TIME`` tform: ``1D``, unit: s
+    * Time stamp of event in instrument specific MJD time reference. See the
+      header keywords ``MJDREFI`` and ``MJDREFF`` for the zero point of
+      the reference time.
+      See also the `OGIP event list`_ standard.
+* ``RA`` tform: ``1E``, unit: deg
+    * Reconstructed event Right Ascension (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
```

Comment:
We have other places in the spec (e.g. pointing table, observation index table) with RA / DEC columns.
OK to move the `See also `HFWG Recommendation R3`_ for the OGIP standard.` sentence to the general section to avoid duplicating info?
http://gamma-astro-data-formats.readthedocs.org/en/latest/info/coordinates.html